test/integration/node/lifecycle_test.go

```diff
@@ -206,19 +198,20 @@ func TestTaintBasedEvictions(t *testing.T) {
 			TerminationGracePeriodSeconds: &gracePeriod,
 		},
 	}
-	tolerationSeconds := []int64{200, 300, 0}
```

Comment:
yeah, this is weird, why wasn't part of the struct before?

Reply:
It seems for 'historical reasons'. I've traced it back to this commit https://github.com/kubernetes/kubernetes/commit/ca10e616bf88ee0f0ed00762c92948761dd56d62. It seems the test was first added as a non-table e2e test. Later it was converted to a table integration test, but the list was not refactored. Seems reasonable to do it now.

Reply:
agreed, it is much better this way